examples/stream_created_from_array_json.ts

```diff
@@ -1,38 +0,0 @@
-import { createClient } from '@clickhouse/client'
```

Comment:
I think in this case stream is pretty much useless and just an array could be used (and we have these examples already)